PartyIcons/View/NameplateSizeMode.cs

```diff
@@ -5,5 +5,6 @@ public enum NameplateSizeMode
         Smaller,
         Medium,
         Bigger,
+        Tiny
```

Comment:
Elements should be ordered in the size, from smallest to the biggest.

Reply:
That one is wrong. These names are synonyms to index numbers, if I place Tiny first, it changes indexes of the rest of items, meaning all setting people used would change too. I. e. Smaller would change to Tiny, Medium to smaller, etc.